raco/myrial/optimizer_tests.py

```diff
@@ -1325,18 +1331,18 @@ def test_galaxy_evolution(self):
         GoI = scan(public:adhoc:X);
         Particles = scan(public:adhoc:Y);
         do
-        Edges = [$0,$1,$2,COUNT(*)] <-
+        Edges = [time,gid1,gid2,COUNT(*) as num] <-
                 [from Particles as P1, Particles as P2, Galaxies
-                where P1.$0 = P2.$0 and P1.$2+1 = P2.$2 and
-                        P1.$2 = Galaxies.$0 and Galaxies.$1 = P1.$1
-                emit P1.$2 as time, P1.$1 as gid1, P2.$1 as gid2, P1.$0];
-        Galaxies = [$0, $1] <-
-          [from GoI emit 1, GoI.$0] +
+                where P1.d = P2.d and P1.f+1 = P2.f and
+                      P1.f = Galaxies.time and Galaxies.gid = P1.e
+                emit P1.f as time, P1.e as gid1, P2.e as gid2];
+        Galaxies = [time, gid] <-
+          [from GoI emit 1 as time, GoI.a as gid] +
           [from Galaxies, Edges
-           where Galaxies.$0 = Edges.$0 and
-           Galaxies.$1 = Edges.$1 and Edges.$3 >= 4
-           emit Galaxies.$0+1, Edges.$2];
-        until convergence async;
+           where Galaxies.time = Edges.time and
+           Galaxies.gid = Edges.gid1 and Edges.num >= 4
+           emit Galaxies.time+1, Edges.gid2 as gid];
+        until convergence async build_EDB;
         store(Galaxies, Galaxies);
         """
         lp = self.get_logical_plan(query, async_ft='REJOIN')
```

Comment:
I can't comment on the definitions of `get_logical_plan()`, `logical_to_physical()`, `compile_to_json()` here, so: don't we need to add a checkbox to MyriaWeb exposing the `async_ft` option? I can create an issue in the `myria-web` repo if so.